sentry-core/src/main/java/io/sentry/core/SendCachedEventFireAndForgetIntegration.java

```diff
@@ -34,25 +36,21 @@ public final void register(@NotNull IHub hub, @NotNull SentryOptions options) {
       return;
     }
 
-    SendFireAndForget sender = factory.create(hub, options);
+    final SendFireAndForget sender = factory.create(hub, options);
 
     try {
-      ExecutorService es = Executors.newSingleThreadExecutor();
-      es.submit(
-          () -> {
-            try {
-              sender.send();
-              options
```

Comment:
We lost this log message. Was that intentional?

Reply:
they now live in SendFireAndForgetEnvelopeSender and SendFireAndForgetEventSender